README.md

```diff
@@ -34,128 +36,207 @@ git clone https://github.com/gnosis/dx-cli.git dx-cli
 
-**4. Make sure the scripts are executable**
+
+**3. Make sure the scripts are executable**
 ```bash
 # Allow the CLI script to be executed
 chmod +x dutchx*
 ```
 
-**5. Network info: Review the list of tokens you want to use**
+**4. Network info: Review the list of tokens you want to use**
+
+> **NOTE**: This step can be skipped if you don't intend to list new tokens on the DutchX.
+
+This step is relevant because the DutchX is an open protocol where anyone can list new tokens to trade. 
 
-For each network, check it's config:
+Each network has the following configuration: 
 * [network-rinkeby.conf](./network-rinkeby.conf)
 * [network-kovan.conf](./network-kovan.conf)
 * [network-mainnet.conf](./network-mainnet.conf)
 
-This step is important, because DutchX ist's just a protocol, anyone can list
-their tokens, every user should add the token that he want to use with the CLI.
-
-Check the complete list of tokens that are added in the `DutchX`:
-* **Rinkeby**: The easiest way to check all token pairs that were added to the
-  DutchX is to check the API:
+Check out the complete list of tokens listed on the `DutchX` here:
+* **Rinkeby**: 
   * [https://dutchx-rinkeby.d.exchange/api/docs/#!/markets/getMarkets](https://dutchx-rinkeby.d.exchange/api/docs/#!/markets/getMarkets)
-* **Mainnet**: The easiest way to check all token pairs that were added to the
-    DutchX is to check the API:
+* **Mainnet**: 
   * [https://dutchx.d.exchange/api/docs/#!/markets/getMarkets](https://dutchx.d.exchange/api/docs/#!/markets/getMarkets)
-* **Kovan**: Unlike `Rinkeby` and `Mainnet`, `Kovan` does't have any API
-  published. To check all available markets, you must do it at smart contract level:
+* **Kovan**: Unlike `Rinkeby` and `Mainnet`, `Kovan` doesn't have a published API. To check all available markets, you must do so on smart contract level:
   * https://dutchx.readthedocs.io/en/latest/smart-contracts_addresses.html
 
-**6. Try the CLI**
+**5. Try the CLI**
 
-Run the `help` command to get a list of all avaliable commands:
+Run the `help` command on the Terminal to get a list of all avaliable commands:
 ```bash
 # Rinkeby
-./dutchx-rinkeby help
+./dutchx-rinkeby -h
 
 # Kovan
-./dutchx-kovan help
+./dutchx-kovan -h
 
 # Mainnet
-./dutchx-mainnet help
+./dutchx-mainnet -h
 ```
 
-# Basic usage
-## Get account balances
-Get balances for all tokens of an account
+# Start Trading 
+## DutchX trading process
+Trading on the DutchX requires you to send your tokens to a smart contract, where the trading occurs. 
+The whole trading process looks as the following image:
+
+<img src="https://raw.githubusercontent.com/gnosis/dx-docs/master/source/_static/DurtchX_graphi.png" width="600" height="400">
+
+
+
+Before we start explaining each step in the image, we will use the balance command to make the process more clear. 
+All the commands used in this explanation will be for the Rinkeby Testnet. If you want to trade on Ethereum Mainnet, change `Rinkeby` to `mainnet`
```

Comment:
I think is more clear:

If you want to trade on Ethereum Mainnet, use `./dutchx-mainnet` instead of `./dutchx-rinkeby`